roles/tempest/files/list_allowed.yml

```diff
@@ -34,3 +34,8 @@ groups:
       - manila_tempest_tests.tests.api
     releases:
       - all
+  - name: ironic-operator
+    tests:
+      - ironic_tempest_plugin.tests.api.admin.*
```

Comment:
we should leave a comment as to why we are only enabling the admin tests

Reply:
@rlandy good catch, so within api tests for ironic_tempest_plugin we have admin, rbac_defaults and we are just running admin instead we should be running all the tests

[ironic api tests](https://github.com/openstack/ironic-tempest-plugin/tree/master/ironic_tempest_plugin/tests/api)